components/typography/Heading.js

```diff
@@ -0,0 +1,42 @@
+
+export default function Heading({
+    typeStyle = "heading-lg",
+    level = 'h2',
+    textColor = 'text-primary-800',
+    className,
+    children
+}) {
+    let classNames
```

Comment:
Please default that var to empty string:

```suggestion
    let classNames = '';
```

Without that if someone will pass not supported `typeStyle` final css class will be render as `<h1 class="... undefined">` - undefined will be render as "undefined" :)

Reply:
Good call! Just committed that suggestion for both :)